contracts/src/Gateway.sol

```diff
@@ -69,6 +69,9 @@ contract Gateway is IGateway, IInitializable {
     // 2. Calling implementation function
     uint256 DISPATCH_OVERHEAD_GAS = 10_000;
 
+    // The maximum fee that can be sent to a destination parachain to pay for execution (DOT)
+    uint128 internal immutable MAX_DESTINATION_TRANSFER_FEE;
+
```

Comment:
Just curious why not make it a storage value like other fee params?

https://github.com/Snowfork/snowbridge/blob/bc62ad1878ff7747594c4153f255c85174522d64/contracts/src/storage/AssetsStorage.sol#L13-L17

Reply:
So originally I did have it as a storage params. But we did not want the functionality of it being able to be updated via the `set_fee_params` extrinsic from the control pallet. Since we didn't need it to be changed at runtime, I decided on just using an immutable for now so that we do not need to have a migration of AssetsStorage.